eth_keys/backends/__init__.py

```diff
@@ -13,6 +13,7 @@
     is_coincurve_available,
 )
 from .native import NativeECCBackend  # noqa: F401
+from .ed25519 import TwistedEdwardsECCBackend
 
 
 def get_default_backend_class() -> str:
```

Comment:
We should make our ed25519 the default for AION

Reply:
Do you reckon I should do it on this branch?

Reply:
I am referring to commit - 7056041ed6f355804a3f996c996ae77f95146e1c

as it solves both the default backend and the address.